src/HealthGPS.Console/model_parser.cpp

```diff
@@ -313,6 +334,9 @@ std::unique_ptr<hgps::RiskFactorModelDefinition>
 load_dynamic_risk_model_definition(const std::string &model_name, const poco::json &opt,
```

Comment:
And the very same comments on the docstring, lowercase and changing to switch here.